lib/cli/invocation/http/invokeEventHandlerUsingHttp.ts

```diff
@@ -63,10 +64,10 @@ export function invokeEventHandlerUsingHttp(location: AutomationClientConnection
                 correlation_id: target.correlationId || await newCliCorrelationId(),
             },
             secrets: (invocation.secrets || []).concat([
-                { uri: "github://user_token?scopes=repo,user:email,read:user", value: process.env.GITHUB_TOKEN },
-                { uri: "github://org_token", value: process.env.GITHUB_TOKEN },
-                { uri: Secrets.OrgToken, value: process.env.GITHUB_TOKEN },
-                { uri: Secrets.UserToken, value: process.env.GITHUB_TOKEN },
+                { uri: "github://user_token?scopes=repo,user:email,read:user", value: credentialsFromEnvironment().token },
```

Comment:
Optional optimization: call `credentialsFromEnvironment()` once, use many times.

Reply:
Actually the results of calling `credentialsFromEnvironment` change over time. First time it is invoked the automation-client isn't up, hence the call to `configurationValue` will fail. I'm not sure what impact this might have on the various usages of the `CredentialsResolver` throughout. That's why I didn't change it to use caching.

Reply:
Ah, I see. yeah, I'll change that.